composer.json

```diff
@@ -10,9 +10,9 @@
   }
   ],
   "require": {
-    "php": "^7.3||^8.0",
-    "drupal/core": "^8.5||^9.1",
-    "symfony/lock": "^3.4||^4.0"
+    "php": "^8.1",
+    "drupal/core": "^9.1",
+    "symfony/lock": "^4.4"
```

Comment:
I confirmed Drupal 9 only supports Symfony 4.x components, so we're good to drop 3.x compatibility.